lib/topology/libraries/manager.py

```diff
@@ -32,7 +32,7 @@
 from collections import OrderedDict
 from argparse import Namespace
 
-from pkg_resources import iter_entry_points
+from importlib.metadata import entry_points
```

Comment:
importlib.metadata does not work in python3.8, you need to use importlib-metadata package to back port it

```suggestion
import sys

if sys.version_info >= (3, 8):
    from importlib.metadata import entry_points
else:
    from importlib_metadata import entry_points  # backport for Python < 3.8
```